packages/babel-plugin-transform-typescript/test/fixtures/enum/boolean-value/output.js

```diff
@@ -1,5 +1,4 @@
 // Not type-correct code
-var E;
-(function (E) {
+var E = /*#__PURE__*/function (E) {
   E[E["A"] = true] = "A";
-})(E || (E = {}));
+}(E || {});
```

Comment:
This IIFE always returns `undefined`, it should return the enum.

Reply:
oops! Thanks!